src/pages/components/tabs/usage.mdx

```diff
@@ -379,18 +401,102 @@ One tab should be selected by default. Users can navigate between tabs by
 pressing right or left arrow keys. For additional keyboard interactions, see the
 [Accessibility tab](https://carbondesignsystem.com/components/tabs/accessibility).
 
-#### Screenreader
+#### Automatic and manual
+
+Automatic and manual tablists differ in how the tab items are activated.
+
+For automatic tablists, focus and selection are synchronized. When the user
+arrows to a tab, it becomes selected, and the tab panel under the tab is updates
+automatically.
+
+Manual tablists allow the user to arrow between the tab items without updating
+the tab panel underneath. When the user navigates using arrows, the selected tab
+remains selected while focus moves to the next tab. In order to select the
+focused tab (and update the tabpanel under the tab), the user would press Enter
+or Space.
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Automatic">
+
+<GifPlayer color='dark'>
```

Comment:
Regarding the gifs: 

— Does the magenta annotation apply to an individual tab at a time? I am wondering if the annotation needs to sit above the tab that is changes or being interacted with in each frame. 

— I also think the gifs should be standalone and go under each individual content section on the page. I think you are right it might be a lot of interaction in the Gatsby tab image component. 

— If you are still experiencing a white border bug with using the Gatsby gif component try reaching out to Thy, she has been working with it recently. However, I think that may be happening because of how the Gatsby tab image component is structured.